ansible-ipi-install/roles/node-prep/tasks/40_bridge.yml

```diff
@@ -137,7 +137,7 @@
       with_items:
         - baremetal
         - "{{ pub_nic }}"
-        - provisioning
+        - {{ prov_bridge }}
```

Comment:
```
- {{ provisioning_bridge | default('provisioning') }}
```

Reply:
this way duplicate the default all over the place, making it hard to maintain in the future
whereas my original solution - you had that value default in one place (and always defined)

Reply:
Hey @yuvalk, however by adding a newly required variable, you will break everyone's existing inventory files as they won't have this change in place. It is better to make the variable optional and default to the value most users are using and just provide the option of changing the value to something different when necessary.

Reply:
I've added a default in defaults/main.yml
so it is not required